plugins/services/src/js/reducers/serviceForm/ExternalVolumes.js

```diff
@@ -119,6 +127,9 @@ module.exports = {
       if (type === SET && `externalVolumes.${index}.containerPath` === joinedPath) {
         state[index].containerPath = value;
       }
+      if (type === SET && `externalVolumes.${index}.size` === joinedPath) {
+        state[index].size= value;
```

Comment:
adding new eslint rule on my list, would you add missing space here and 3 lines below?